README.md

```diff
@@ -110,3 +110,19 @@ Cron Expression: Cron expressions are used for more fine-tuned alert scheduling.
 Expires: This refers to how long an alert is displayed if it triggered, and not how long the alert itself is valid.
 
 Trigger Conditions: These can be used to set some additional behavior on alerts, such as how many events have to occur before an alert is triggered, if an alert should trigger once or once for each result, how much time should elapse before an alert can trigger again. 
+
+## Debugging
+
+Useful tools for testing implementation and debugging issues.
+
+### Postman
+
+There is a postman collection and environment included in the test folder. This collection can be used to test and confirm the implementation of the notification service.
```

Comment:
maybe do the [test](url_to_test) for handy dandy quick linking

Reply:
[]()

Reply:
borf

Reply:
Wat